### PR TITLE
Maintainer talk for SciPy2022

### PR DESCRIPTION
**Pull request by tupui**
Draft for SciPy2022's conference. Focus on the maintainer side of things with current and future work on the architecture/support to maintainers of other projects.

### CODE REVIEW

**Comment by tupui** [comment]
Either here or during the talk, it would be good to make the link with the past presentation/proposal https://github.com/scientific-python/scientific-python-archive/blob/main/presentations/accepted/scipy2021/specs/proposal.md

**Comment by stefanv** [comment]
I pushed some edits.

File: presentations/proposed/scipy2022/maintainers/proposal.md [hunk 1]
```diff
@@ -5,35 +5,33 @@ Jul 11-17, 2022
...
-Your placement in the program will be based on reviews of your abstract.
-This should be a roughly 500 word outline of your presentation. This outline
-should concisely describe software of interest to the SciPy community, tools or
-techniques for more effective computing, or how scientific Python was applied
-to solve a research problem. A traditional background/motivation, methods,
-results, and conclusion structure is encouraged but not required. Links to
-project websites, source code repositories, figures, full papers, and evidence
-of public speaking ability are encouraged.
--->
+The Scientific Python project's vision is to help pave the way towards a unified, expanded scientific Python community. It focuses its efforts along two primary axes: _(i)_ to create a joint community around all scientific projects and (ii)_to support maintainers by building cross-cutting technical infrastructure and tools. This talk will focus on the second aspect, presenting current and future initiatives.
```
Comment (rossbar) [inline presentations/proposed/scipy2022/maintainers/proposal.md:19]:
```suggestion
The Scientific Python project's vision is to help pave the way towards a unified, expanded scientific Python community. It focuses its efforts along two primary axes: _(i)_ to create a joint community around all scientific projects and _(ii)_to support maintainers by building cross-cutting technical infrastructure and tools. This talk will focus on the second aspect, presenting current and future initiatives.
```
Formatting nit

**Review by rossbar — COMMENTED** [review]
This looks like a great submission to me - just a minor md formatting nit while reading through.

It may be too much detail for the abstract, but you could mention some of the ongoing cross-project technical discussions such as the API dispatch discussion or the SPEC about lazy loading, which might be of particular interest on the maintainers track. I think the abstract is totally fine without these details, but just a thought in case you were looking to expand on something!

**Comment by tupui** [comment]
Thanks everyone

**Pull request merged**